libs/extract/src/lib/form.ts

```diff
@@ -6,3 +6,8 @@ export interface IExpandableInformation {
   expandableInfo?: ReactNode
   expandableInfoButtonLabel?: string
 }
+
+export interface ILableAndLableInformation {
```

Comment:
There's a spelling error in the interface name I believe, "lable" should probably be "label".